docs/cookbooks/common_workflows.md

```diff
@@ -86,31 +86,49 @@ SELECT
   -- format:on
 ```
 
-## Add a new field to clients_daily
-
-Adding a new field to `clients_daily` also means that field has to propagate to several
-downstream tables, so this is a more complex case.
-
-1. Open the `clients_daily_v6` `query.sql` file and add new field definitions
-1. Run `./bqetl query validate telemetry_derived.clients_daily_v6` to dry run and format the query
-1. Run `./bqetl query schema update telemetry_derived.clients_daily_v6 --update_downstream` to make local `schema.yaml` updates and update schemas of downstream dependencies
-    * This requires GCP access
-    * `schema.yaml` files of downstream dependencies, like `clients_last_seen_v1`, will be automatically updated
-1. Open PR with changes
-    * The `dry-run-sql` task is expected to fail at this point due to mismatch with deployed schemas
-1. PR reviewed and approved
+## Add a new field to a table schema
+Adding a new field to a table schema also means that the field has to propagate to several downstream tables, which makes it a more complex case.
+
+1. Open the query.sql file inside the `<dataset>.<table>` location and add the new field definitions.
+1. Run `./bqetl query validate <dataset>.<table>` to dry run and format the query.
+1. Run `./bqetl query schema update <dataset>.<table> --update_downstream` to make local schema.yaml updates and update schemas of downstream dependencies.
+   * [x] This requires [GCP access](https://docs.telemetry.mozilla.org/cookbooks/bigquery/access.html#bigquery-access-request).
+   * [x] Note that schema.yaml files of downstream dependencies will be automatically updated.
+1. Open a new PR with these changes.
+1. The dry-run-sql task is expected to fail at this point due to mismatch with deployed schemas!
+1. PR reviewed and approved.
+1. Deploy schema changes by running: `./bqetl query schema deploy <dataset>.<table>;`
+1. Rerun the CI pipeline in the PR.
+   *[x] Make sure all dry runs are successful.
+1. Merge pull-request.
+
+The following is an example to update a new field in telemetry_derived.clients_daily_v6
+
+### Example: Add a new field to clients_daily
```

Comment:
I'm happy you kept a practical example. 🙇  Sometimes I find it helps me to understand what the input is meant to be.